torchtext/datasets/text_classification.py

```diff
@@ -71,9 +72,16 @@ def __init__(self, vocab, data, labels):
         """Initiate text-classification dataset.
 
         Arguments:
-            url: url of the online raw data files.
-            root: Directory where the dataset are saved.
-            ngrams: a contiguous sequence of n items from s string text.
+            vocab: Vocabulary object used for dataset.
+            data: a list of label/tokens tuple. tokens are a tensor after
+                numericalizing the string tokens. label is an integer.
+                [(label1, tokens1), (label2, tokens2), (label2, tokens3)]
+            label: a set of the labels.
+                {label1, label2}
+
+        Examples:
+            See _setup_datasets() function in torchtext.datasets.text_classification
```

Comment:
Since this is internal (due to the leading underscore) we can't use it as an example :/